test_unstructured/partition/test_html_partition.py

```diff
@@ -267,27 +267,27 @@ def test_partition_html_raises_with_too_many_specified():
 
 def test_partition_html_on_ideas_page(filename="example-docs/ideas-page.html"):
     elements = partition_html(filename=filename)
-    assert len(elements) == 4
+    assert len(elements) == 1
+
+    assert elements[0] == Table(
+        text="<table>\n<tbody>\n<tr><td>January 2023</td><td>(</td><td>Someone"
+        "</td><td>fed my essays into GPT to make something that could answer"
+        "\nquestions based on them, then asked it where good ideas come from.  "
+        "The\nanswer was ok, but not what I would have said. This is what"
+        " I would have said.) </td><td>The way to get new ideas is to notice"
+        " anomalies: what seems strange,\nor missing, or broken? You can see"
+        " anomalies in everyday life (much\nof standup comedy is based on this), "
+        "but the best place to look for\nthem is at the frontiers of knowledge. "
+        "</td><td>Knowledge grows fractally.\nFrom a distance its edges look smooth,"
+        " but when you learn enough\nto get close to one, you&#x27;ll notice "
+        "it&#x27;s full of gaps. These gaps\nwill seem obvious; it will seem "
+        "inexplicable that no one has tried\nx or wondered about y. In the best case,"
+        " exploring such gaps yields\nwhole new fractal buds. </td></tr>\n</tbody>\n</table>",
```

Comment:
the convention for `Table` elements is for `.text` to just be concatentated text, but `metadata.text_as_html` to have the html string, as above, so it looks like the html parser should be updated to reflect that convention.

Reply:
Ahhh makes sense 👍🏻